File: appserver-legacy/src/main/java/org/radarbase/appserver/exception/MessageTransmitException.java
package org.radarbase.appserver.exception;

public class MessageTransmitException extends Exception {
    private static final long serialVersionUID = -281834508766939L;

    public MessageTransmitException(String message) {
        super(message);
    }
    public MessageTransmitException(String message, Throwable e) {
        super(message, e);
    }
}

File: appserver-legacy/src/main/kotlin/org/radarbase/appserver/exception/MessageTransmitException.kt
package org.radarbase.appserver.exception

import org.radarbase.appserver.util.OpenClass
import java.io.Serial

@OpenClass
class MessageTransmitException : Exception {
    companion object {
        @Serial
        private const val serialVersionUID = -281834508766939L
    }

    constructor(message: String) : super(message)
    constructor(message: String, e: Throwable) : super(message, e)
}
